submissions/YanaGushlevskaya/html-css-popup/dist/index.html

```diff
@@ -0,0 +1,320 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>
+    
+  Pop-up menu
+
+  </title>
+  <link rel="stylesheet" href="css/main.min.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+  <link rel="shortcut icon" type="image/png" href="img/kottans.png"/>
```

Comment:
No images are displayed in your menu. Fix it

Reply:
Thank you for your review! I fixed it: a problem was with image names (in html file they were uppercase, but in resources folder - lowercase). Please, view one more time when you'll have time.